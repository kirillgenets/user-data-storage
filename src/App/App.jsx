import logo from './logo.svg';
import './App.css';
import Form from '../components/Form';
import DataTable from '../components/DataTable';

function App() {
  return (
    <div className="App">
      <Form />
      <DataTable />
    </div>
  );
}

export default App;
