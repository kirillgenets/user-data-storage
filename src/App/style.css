.app {
  display: flex;
  flex-direction: column;
  gap: 40px;

  padding: 30px;
}
