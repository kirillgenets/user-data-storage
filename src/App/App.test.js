import { shallow } from 'enzyme';

import App from './App';

describe('App', () => {
  it('initial test', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
    expect(component.find('a')).toHaveLength(1);
  });
});
