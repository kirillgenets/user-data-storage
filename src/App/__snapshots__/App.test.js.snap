// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App initial test 1`] = `ShallowWrapper {}`;
